packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1148,54 +1153,52 @@ enum FocusHighlightStrategy {
 
 /// Manages the focus tree.
 ///
-/// The focus tree keeps track of which [FocusNode] is the user's current
-/// keyboard focus. The widget that owns the [FocusNode] often listens for
-/// keyboard events.
-///
-/// The focus manager is responsible for holding the [FocusScopeNode] that is
-/// the root of the focus tree and tracking which [FocusNode] has the overall
-/// focus.
+/// The focus manager is responsible for tracking which [FocusNode] has the
+/// primary input focus (the [primaryFocus]), holding the [FocusScopeNode] that
+/// is the root of the focus tree (the [rootScope]), and what the current
+/// [highlightMode] is.
 ///
 /// The [FocusManager] is held by the [WidgetsBinding] as
 /// [WidgetsBinding.focusManager], and can be conveniently accessed using the
-/// [focusManager] global accessor.
+/// [FocusManager.instance] static accessor.
 ///
 /// To find the [FocusScopeNode] for a given [BuildContext], use
 /// [FocusScope.of].
 ///
 /// The [FocusManager] knows nothing about [FocusNode]s other than the one that
-/// is currently focused (accessible via the [primaryFocus] global accessor). If
-/// a [FocusScopeNode] is removed, then the [FocusManager] will attempt to focus
-/// the next [FocusScopeNode] in the focus tree that it maintains, but if the
-/// current focus in that [FocusScopeNode] is null, it will stop there, and no
-/// [FocusNode] will have focus.
+/// is currently focused (accessible via the [primaryFocus] global accessor).
 ///
 /// If you would like notification whenever the [primaryFocus] changes, register
-/// a listener with [addListener]. When you no longer want to receive events, as
-/// when your object is about to be disposed, you must unregister with
-/// [removeListener] to avoid memory leaks. Removing listeners is typically done
-/// in [State.dispose] on stateful widgets.
+/// a listener with [addListener]. When you no longer want to receive these
+/// events, as when your object is about to be disposed, you must unregister
+/// with [removeListener] to avoid memory leaks. Removing listeners is typically
+/// done in [State.dispose] on stateful widgets.
 ///
 /// The [highlightMode] changes are notified separately via
 /// [addHighlightModeListener] and removed with [removeHighlightModeListener].
 ///
 /// See also:
```

Comment:
This looks like a whole lot fo See Also's now. Is there a way to narrow the focus (pun may or may not be intended 😆) so it is less overwhelming to someone unfamiliar with focus? As in, is there a logical progression of topics we can string together as each one is visited rather than the whole bunch at once?

Reply:
Nice!